pkg/net/libp2p/libp2p.go

```diff
@@ -58,6 +58,12 @@ const (
 	ConnectedPeersCheckTick = time.Minute * 1
 )
 
+// Keep Network protocol identifiers
+const (
+	ProtocolBeacon = "keep-beacon"
+	ProtocolECDSA  = "keep-ecdsa"
```

Comment:
Yes, `keep-ecdsa` imports entire libp2p. I placed it here because the entire network layer is in `keep-core`, there is even no `net` package in `keep-ecdsa`.